pages/common/ansiweather.md

```diff
@@ -0,0 +1,15 @@
+# ansiweather
+
+> AnsiWeather is a Shell script for displaying the current weather conditions in your terminal.
+
+- Display forecast using metric units for the next five days for Rzeszow, Poland:
```

Comment:
```suggestion
- Display a forecast using metric units for the next five days for Rzeszow, Poland:
```

Perhaps this will read better? Same goes for the other examples.